src/cplscheme/tests/ExplicitCouplingSchemeTest.cpp

```diff
@@ -328,14 +329,17 @@ BOOST_AUTO_TEST_CASE(testConfiguredSimpleExplicitCoupling)
   std::string configurationPath(_pathToTests + "explicit-coupling-scheme-1.xml");
   std::string nameParticipant0("Participant0");
   std::string nameParticipant1("Participant1");
+  int         dimensions = 3;
 
   xml::XMLTag          root = xml::getRootTag();
   PtrDataConfiguration dataConfig(new DataConfiguration(root));
-  dataConfig->setDimensions(3);
+  dataConfig->setDimensions(dimensions);
   PtrMeshConfiguration meshConfig(new MeshConfiguration(root, dataConfig));
-  meshConfig->setDimensions(3);
-  m2n::M2NConfiguration::SharedPointer m2nConfig(new m2n::M2NConfiguration(root));
-  CouplingSchemeConfiguration          cplSchemeConfig(root, meshConfig, m2nConfig);
+  meshConfig->setDimensions(dimensions);
+  m2n::M2NConfiguration::SharedPointer         m2nConfig(new m2n::M2NConfiguration(root));
+  precice::config::PtrParticipantConfiguration participantConfig(new precice::config::ParticipantConfiguration(root, meshConfig));
+  participantConfig->setDimensions(dimensions);
+  CouplingSchemeConfiguration cplSchemeConfig(root, meshConfig, m2nConfig, participantConfig);
```

Comment:
Yes and no. I agree that this will help us to make this PR here less complex. However, adding the `ParticipantConfiguration` is only required because of the changes in the configuration w.r.t the experimental tag introduced here.

I think it's still a good idea to use valid configuration (Meaning: Files including one or more Participants) files in our tests that require a configuration file. 

I think taking a look at https://github.com/precice/precice/pull/1215 probably explains this better.

Reply:
#1215 is merged and this resolves this.